docs/samples/custom/torchserve/logger/logger_demo.ipynb

```diff
@@ -0,0 +1,364 @@
+{
```

Comment:
<p>port and component-port is not exposed on the yaml spec, we can actually fix the logger port to use an uncommon port number </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kubeflow/kfserving/pull/1228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>